defaults/main.yml

```diff
@@ -14,8 +14,8 @@ ssl_cert_path: /etc/pki/tls/certs
 ssl_key_path: /etc/pki/tls/private
 
 # UI
-cas_screen_welcome_instructions: Enter your Username and Password
-cas_screen_welcome_security: For security reasons, please Log Out and Exit your web browser when you are done accessing services that require authentication!
+cas_screen_welcome_instructions: Login with your OUNetID (4x4)
```

Comment:
No need to change the default values.  These message defaults are set to the cas out of the box defaults.  We can just set the values appropriately in production. If you have bits of text you want to set, define a var with the original value here, and then we'll set the vars on deploy.